cg/meta/workflow/analysis.py

```diff
@@ -207,20 +207,34 @@ def get_analysis_finish_path(self, case_id: str) -> Path:
 
     def add_pending_trailblazer_analysis(self, case_id: str) -> None:
         self.check_analysis_ongoing(case_id)
+        application_type: str = self.get_application_type(
+            self.status_db.get_case_by_internal_id(case_id).links[0].sample
+        )
+        config_path: str = self.get_job_ids_path(case_id).as_posix()
+        data_analysis: str = str(self.workflow)
+        email: str = environ_email()
+        order_id: str = str(self._get_order_id_from_case_id(case_id))
+        out_dir: str = self.get_job_ids_path(case_id).parent.as_posix()
+        slurm_quality_of_service: str = self.get_slurm_qos_for_case(case_id)
+        ticket: str = self.status_db.get_latest_ticket_from_case(case_id)
+        workflow_manager: str = self.get_workflow_manager()
         self.trailblazer_api.add_pending_analysis(
```

Comment:
This is basically extracting the nested calls and putting stuff in alphabetical order. `order_id` should be the only thing differing functionality-wise.